include/algorithms/math/abs.h

```diff
@@ -89,15 +96,19 @@ class DAAL_EXPORT Batch : public daal::algorithms::Analysis<batch>
     typedef algorithms::math::abs::Input  InputType;
     typedef algorithms::math::abs::Result ResultType;
 
-    /** Default constructor */
-    Batch()
+    /**
+     * Default constructor
+     * \DAAL_DEPRECATED
+     */
+    DAAL_DEPRECATED Batch()
     {
         initialize();
     }
 
     /**
      * Constructs the absolute value function by copying input objects of another absolute value function
      * \param[in] other function to be used as the source to initialize the input objects of the absolute value function
+     * \DAAL_DEPRECATED_USE{ cloneImpl() }
```

Comment:
we have to recommend interface2::Batch instead

Reply:
fixed